pkg/restapi/issuer/operation/operations.go

```diff
@@ -570,8 +570,12 @@ func (o *Operation) createIssuerProfile(pr *ProfileRequest) (*vcprofile.DataProf
 }
 
 // createIssuerProfileVault creates the vault associated with the profile
-func (o *Operation) createIssuerProfileVault() (string, error) {
-	vaultLocationURL, err := o.edvClient.CreateDataVault(&models.DataVaultConfiguration{ReferenceID: uuid.New().String()})
+func (o *Operation) createIssuerProfileVault(did string) (string, error) {
+	dataVaultConfig := &models.DataVaultConfiguration{Sequence: 0, Controller: did, ReferenceID: uuid.New().String(),
+		KEK:  models.IDTypePair{ID: "https://example.com/kms/12345", Type: "X25519KeyAgreementKey2019"},
```

Comment:
`https://example.com/kms/12345` - Is this hardcoding expected ?

Reply:
it's just a hardcoded filler and the vault configuration is not used in edv right now.

Reply:
Oh ok. If we need to fill the URI, then we can generate one dynamically using `uuid.New().URN()`.